src/libcore/iter/adapters/fuse.rs

```diff
@@ -201,7 +172,131 @@ macro_rules! unchecked {
 }
```

Comment:
This `unchecked!` macro is now defined far from its use, and those uses are also scattered more with how the `Impl` specializations are mixed. Let's move this up next to `fuse!` so at least the macros are organized together.

Reply:
Done